modules/ROOT/pages/_partials/sync/js-client-offline-and-conflict.adoc

```diff
@@ -47,6 +47,28 @@ In the following example, the app will perform an `ADD_TASK` mutation which will
 
 For more information, see link:https://www.apollographql.com/docs/react/essentials/mutations.html#props[Apollo's document about mutations].
 
+=== Global Update Functions
+
+Apollo client holds all mutation parameters in memory.
+An offline Apollo client will continue to store mutation parameters and once online, it will restore all mutations to memory. Any Update Functions that are supplied to mutations cannot be cached by an Apollo client resulting in the loss of all optimisticResponses after a restart. _Update functions_ supplied to mutations cannot be saved in the cache. 
+As result all _optimisticResponses_ will disappear from application after restart and it will appear back only when application becomes online and they will be replicated to server.
```

Comment:
"As a result, all _optimisticResponses_ will disappear from the application after a restart and it will only reappear  when the Apollo client becomes online and successfully syncs with the server."